remsfal-services/remsfal-notification/src/main/java/de/remsfal/notification/boundary/MailConsumerResource.java

```diff
@@ -0,0 +1,38 @@
+package de.remsfal.notification.boundary;
+
+import de.remsfal.core.api.AddressEndpoint;
+import de.remsfal.core.json.AddressJson;
+import de.remsfal.core.json.CountryListJson;
+import de.remsfal.core.json.UserJson;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import org.eclipse.microprofile.reactive.messaging.Incoming;
+import org.jboss.logging.Logger;
+
+import java.util.List;
+
+/**
+ * @author Alexander Stanik [alexander.stanik@htw-berlin.de]
+ */
+
+@ApplicationScoped
+public class MailConsumerResource implements AddressEndpoint {
```

Comment:
Die Vererbung von einem Endpoint ist unnötig.

```suggestion
public class MailConsumerResource {
```